CodeTrans/microservice/gaudi/README.md

```diff
@@ -0,0 +1,88 @@
+# Build Mega Service of CodeTrans on Gaudi
+
+This document outlines the deployment process for a CodeTrans application utilizing the [GenAIComps](https://github.com/opea-project/GenAIComps.git) microservice pipeline on Intel Gaudi server. The steps include Docker image creation, container deployment via Docker Compose, and service execution using microservices `llm`. We will publish the Docker images to Docker Hub soon, it will simplify the deployment process for this service.
+
+## 🚀 Build Docker Images
+
+First of all, you need to build Docker Images locally and install the python package of it. This step can be ignored after the Docker images published to Docker hub.
+
+### 1. Source Code install GenAIComps
+
+```bash
+git clone https://github.com/opea-project/GenAIComps.git
+cd GenAIComps
+```
+
+### 2. Build the LLM Docker Image with the following command
+
+```bash
+docker build -t opea/gen-ai-comps:llm-tgi-gaudi-server --no-cache --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f comps/llms/langchain/docker/Dockerfile .
+```
+
+Then run the command `docker images`, you will have the following Docker Image:
+
+- `opea/gen-ai-comps:llm-tgi-server`
+
+### 3. Build MegaService Docker Image
+
+```bash
+docker build -t opea/gen-ai-comps:codetrans-megaservice-server --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f docker/Dockerfile .
```

Comment:
git clone https://github.com/opea-project/GenAIExamples
cd GenAIExamples/CodeTrans/microservice/gaudi/  

where is the dockerfile?